packages/cli/src/actions/dev-plugin.ts

```diff
@@ -27,7 +26,7 @@ const devPlugin: DevPluginCommandHandler = async ({ type, name }) => {
   try {
     dirname = path.join(process.cwd(), name);
     if (fse.existsSync(dirname)) {
-      spinner.fail(`a directory or file called ${name} already exists. Please use a new working directory`);
+      logFail(`a directory or file called ${name} already exists. Please use a new working directory`);
```

Comment:
As I commented before, we can use `abort` instead of the above 2 lines which calls `process.exit` internally.